docs/user-docs/generate-bundle.rst

```diff
@@ -0,0 +1,29 @@
+generate-bundle.sh
```

Comment:
This information on this page already exists on the "Usage" page, rather than have a new page just for this script maybe better to update the existing info to cover anything extra added here and then reference the Usage page in your Tools page?